tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -4338,6 +4363,15 @@ wan/lacp/test_wan_lag_min_link.py::test_lag_min_link:
       - "len(minigraph_portchannels) < 2"
       - "not any(len(v['members']) > 1 for _, v in minigraph_portchannels.items())"
 
+#######################################
+#####             wol            #####
+#######################################
+wol:
+  skip:
+    reason: "Not supported on this DUT topology"
+    conditions:
+      - topo_type not in ['mx', 'm0']
```

Comment:
@bachalla every condition should have ""

Please double check if all conditions have ""